src/runtime/image.ts

```diff
@@ -106,6 +114,22 @@ async function getMeta (ctx: ImageCTX, input: string, options: ImageOptions) {
   return meta
 }
 
+async function fetchPayload (page, context) {
+  if (!context.nuxtState) {
+    context.beforeNuxtRender(({ nuxtState }) => {
+      pagePayload = nuxtState.data[0]
+    })
+  } else {
+    try {
+      // @ts-ignore
+      const payload = await window.__NUXT_IMPORT__(decodeURI(page.path.replace(/\/$/, '')), encodeURI(context.nuxtState.staticAssetsBase + page.path + 'payload.js'))
```

Comment:
We should use `getStaticAssetsPath` for generating payload path, `getStaticAssetsPath` will handle the router base and generates correct path.
Checkout https://github.com/nuxt/nuxt.js/blob/dev/packages/vue-app/template/App.js#L313

Reply:
done